Microsoft.DotNet.Interactive.PowerShell/Modules/Microsoft.DotNet.Interactive.PowerShell/Microsoft.DotNet.Interactive.PowerShell.psd1

```diff
@@ -48,7 +48,11 @@ NestedModules = @('Microsoft.DotNet.Interactive.PowerShell.dll')
 FunctionsToExport = @()
 
 # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
-CmdletsToExport = @('Trace-PipelineObject')
+CmdletsToExport = @(
+    'Get-JupyterHtml',
+    'Invoke-JupyterJavaScript',
```

Comment:
not sure about the ```Invoke``` verb here, is more an ```Eval```/```Run```. @jonsequitur ?

Reply:
Invoke seemed closest on the [list of approved PowerShell verbs](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7).

Reply:
Updated to `Use-JavaScript`